python/cugraph/cugraph/community/louvain.py

```diff
@@ -89,6 +93,17 @@ def louvain(G, max_level=None, max_iter=None, resolution=1.0, threshold=1e-7):
 
     """
 
+    # FIXME: Onece the graph construction calls support isolated vertices through
+    #  the C API (the C++ interface already supports this) then there will be
+    # no need to compute isolated vertices here.
+
+    isolated_vertices = list()
+    if is_nx_graph_type(type(G)):
+        isolated_vertices = [v for v in range(G.number_of_nodes()) if G.degree[v] == 0]
+    else:
+        # FIXME: Gather list of isolated vertices of G (of cugraph.Graph type)
```

Comment:
I don't understand this `FIXME`. THere is currently only one way to pass isolated vertices to our API and it is only through networkx so I don't see why the else statement is necessary here. Later once we start supporting isolated vertices it wouldn't be through this else statement. Unless I am missing something

Reply:
The point I am trying to make is that when we start supporting isolated vertices with a cuGraph Graph, I don't think it will be through this `else `statement therefore I find it unnecessary.

Reply:
>There is currently only one way to pass isolated vertices to our API

Actually a user can pass isolated vertices to our API by creating a cuGraph Graph from an adjacency list. In this case the `else` statement will be useful if you decide to add support for that particular case in this PR.